ui/dashboards/src/components/Variables/variable-model.ts

```diff
@@ -101,3 +101,25 @@ export const VARIABLE_TYPES = [
   { label: 'List', kind: 'ListVariable' },
   { label: 'Text', kind: 'TextVariable' },
 ] as const;
+
+/*
+ * Check whether saved variable definitions are out of date with current default list values in Zustand store
+ */
+export function checkSavedVariablesStatus(definitions: VariableDefinition[], varState: VariableStateMap) {
```

Comment:
oh wait i noticed this is duplicated, does this need to be removed?

Reply:
Yeah I’ll remove this!